src/views/bovine/ListBatch.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div class="text-center">
+    <div class="col-12 table-responsive">
+      <div style="margin-top:25px;">
+        <h3>Listado Lotes</h3>
+      </div>
+      <table class="table">
+
+        <thead>
+        <tr v-if="batches.length !== 0">
+          <th scope="col">Nombre</th>
+          <th scope="col">Descripcion</th>
+          <th class="text-end" scope="col">Acciones</th>
+        </tr>
+        <tr v-else>
+          <th scope="col">No se encontraron resultados para las condiciones de búsquedas ingresadas.</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr v-for="(batch,index) in batches" :key="batch.batchName">
+          <td>{{ batch.batchName }}</td>
+          <td>{{ batch.description }}</td>
+          <td class="text-end">
+            <font-awesome-icon
+                class="me-2" icon="edit"
+                style="cursor:pointer;font-size:20px;"
+                title="Editar bovinos del lote"
+                v-on:click="openDeleteBatchModal(batch)">
+            </font-awesome-icon>
+            <font-awesome-icon
+                icon="trash"
+                style="cursor:pointer;font-size:20px;"
+                title="Eliminar lote"
+                v-on:click="formDeleteBatch(batch.batchName,index)">
+            </font-awesome-icon>
+          </td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+  <confirmation-modal
+      :confirmation-message="'¿Confirma que desea eliminar el lote '
+      + this.deleted['name'] + '?'"
+      modal-id="DeleteModal" title="Eliminar"
+      @acceptModal="modalDelete(); this.deleteModal.hide()" @rejectModal="this.deleteModal.hide(); this.deleted = {}"></confirmation-modal>
```

Comment:
this.deleteModal.hide() lo sacaria de aca, lo pondria directamente en modalDelete() y solo si salio todo bien.